maquetado/scss/books.scss

```diff
@@ -1,10 +1,12 @@
 .book-container {
+  display: grid;
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-05;
-  display: flex;
+  grid-template-columns: 0.4fr 0.6fr;
   padding: 20px;
   max-width: 1000px;
   margin: 60px auto;
```

Comment:
No need to use this margin here. You can just do `align-self: center` to do a vertical alignment when you are using `display: grid` in this element's parent